tests/data/preview_context_managers/auto_detect/features_3_8.py

```diff
@@ -0,0 +1,18 @@
+# This file doesn't used any Python 3.9+ only grammars.
```

Comment:
Can you test that something like `with (a): pass` doesn't get autodetected as 3.9+?

Reply:
Done